### PR TITLE
#157045480 Merge gh-pages with master 

### PR DESCRIPTION
**Pull request by Davincier**
Merging the gh-pages with the master to develop and integrate UI pages

### CODE REVIEW

**Comment by Davincier** [comment]
Pull request for UI development

**Comment by SrMoffat** [comment]
Hey bro,
Good to see you are making progress man! Several things to point out:
a)	Remember to update your README file to be slightly more descriptive than it is
b)	Include a link to your github pages so it can be easy to access the UI and view it
c)	In your order.html, in the table, use a differently colored font because the text is not visible
Keep pushing bro #YOYO

File: README.md [hunk 1]
```diff
@@ -2,3 +2,7 @@
 Cohort 27 - Andela
 This is a project to book meals in a hotel, with ability of adding multiple user and real time change of events on the web application.
 It combines html, css and Js so far
+Creating a web application, to fit meals ordering market specificaion
+following are my handles:
+https://www.pivotaltracker.com/n/projects/2167130
+https://github.com/Davincier/Book-A-Meal
```
Comment (kwahalf) [inline README.md]: add screenshots of your app and a description of what your app does, plus explain to a new user who wants  to use your app how to navigate around your app

**Review by kwahalf — CHANGES_REQUESTED** [review]
And your commit messages should follow the best practices

**Comment by kwahalf** [comment]
@Davincier  fix the merge conflicts

**Comment by Davincier** [comment]
#157080994 Merging the gh-pages with the master readme update

**Pull request merged**